x/bank/keeper/keeper.go

```diff
@@ -18,6 +18,8 @@ import (
 
 var _ Keeper = (*BaseKeeper)(nil)
 
+const govModuleName = "gov"
```

Comment:
you can use this https://github.com/provenance-io/cosmos-sdk/blob/d13d488d9b0e9e32dbf712707cd637536976f197/x/gov/types/keys.go#L12-L24 

```
const (
	// ModuleName is the name of the module
	ModuleName = "gov"
```
avoid duplicate const